### DISCUSSION TITLE
Tiling Errors on  WebRAVE 18030002 TauDEM has tiling issues

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/riverscapes-website-jekyll#48] [Potential Bug?]
These errors were reported by @nbouwes  on [this TauDEM 18030002 project](https://webrave.riverscapes.xyz/#/Anabranch/db7a3e90-9e40-41ea-b708-f0d637efbd81), which was for the Kern and based on LiDAR inputs.

It is the LiDAR TauDEM and has Tiling Errors on only the TWI and Flow Accumulation the tiling broke on:
![image](https://user-images.githubusercontent.com/9723905/164027280-e4efd8e1-dd61-4256-834d-ad2c330a9d65.png)

**Comment by joewheaton** [comment]
@joewheaton replied to @nbouwes 

> Thanks Nick,
> Yes the WebRAVE view of the LiDAR based projects is failing on a bunch of TauDEM and a couple VBET projects. This is a known issue and one we have to make some headway on soon. If you download the project I suspect it is working fine. We have to get zoom tiling levels figured out for reach-scale projects and a strategy for these big LiDAR datasets. Matt anything to add or is this one any different?

@MattReimer suggested:

> I'd have to dig into it to find out why it's happening. 
> 
> For rasters I've seen it fail on invalid values, big datasets that max out the tilerizer's memory. There should be logs.

**Comment by MattReimer** [comment]
Ok, I did some really cursory looking into it and this is a weird raster. It's 2.2Gb and it's got some really weird asymptotic values in it. None of that is definitive but I'm putting this here just to remind myself later. 

Unfortunately I need to sideline this fix for now so I can get the new warehouse online. We can revisit if this is still a problem in the new system.

```
tiles/Anabranch/TauDEM/db7a3e90-9e40-41ea-b708-f0d637efbd81/intermediates/areadinf_sca.tif/index.json
```

**Reply by joewheaton** [reply]
All sounds good... not a high priority, just queuing up so it is addressed as part of Warehouse V2 refactor later.
